trussworks/react-uswds
# fix: Sass modules bug

# Summary

Adds required USWDS paths for webpack in both main app config and storybook config to support USWDS tokens and theme variables inside Sass modules.

## Related Issues or PRs



Closes #2554 

## How To Test





1. Add a `*.module.scss` file for a component
2. Import the styles into the component
3. Use the USWDS syntax for forwarding the theme tokens and utilities
4. See results of styles in storybook

### Screenshots (optional)
